jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.kt

```diff
@@ -226,5 +246,6 @@ class AwsSettingsConfigurable(private val project: Project) : SearchableConfigur
     companion object {
         private const val CLOUDDEBUG = "clouddebug"
         private const val SAM = "sam"
+        private const val awsCli = "awsCli"
```

Comment:
please use AWS_CLI to follow naming conventions of our consts

Reply:
changed the variable name to AWS_CLI